### PR TITLE
init UI edit screen

### PR DESCRIPTION
**Pull request by thaohq-0814**
Init UI edit screen
Link: https://edu-redmine.framgia.vn/issues/6632
image

### CODE REVIEW

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java [hunk 1]
```diff
@@ -57,18 +59,19 @@ public void onStop() {
     }
 
     void setUp() {
+        Navigator navigator = new Navigator(this);
         ImageRepository repository =
                 ImageRepository.getInstance(ImageRemoteDataSource.getInstance(),
                         ImageLocalDataSource.getInstance(new SharedPrefsApi(
                                 Objects.requireNonNull(getActivity()).getApplicationContext())));
-        mPhotoDetailViewModel = new PhotoDetailViewModel(repository);
+        mPhotoDetailViewModel = new PhotoDetailViewModel(repository, navigator);
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java]:
```suggestion
        mPhotoDetailViewModel = new PhotoDetailViewModel(repository, new Navigator(this););
```

File: app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailViewModel.java [hunk 2]
```diff
@@ -49,4 +55,10 @@ public void accept(Throwable throwable) throws Exception {
     public void setPhoto(Photo photo) {
         mPhotoObservableField.set(photo);
     }
+
+    public void onEditButtonClicked(View view) {
+        mNavigator.goNextChildFragment(null, R.id.layoutContainer,
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailViewModel.java]: phần này bên view e

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**